http/cookie_test.ts

```diff
@@ -177,20 +177,14 @@ Deno.test({
       "Space=Cat; Secure; HttpOnly; Max-Age=2",
     );
 
-    let error = false;
     res.headers = new Headers();
-    try {
-      setCookie(res, {
-        name: "Space",
-        value: "Cat",
-        httpOnly: true,
-        secure: true,
-        maxAge: 0,
-      });
-    } catch {
-      error = true;
-    }
-    assert(error);
+    setCookie(res, {
+      name: "Space",
+      value: "Cat",
+      httpOnly: true,
+      secure: true,
+      maxAge: 0,
+    });
 
```

Comment:
I think we should also add this test 
```suggestion

let error = false;
    res.headers = new Headers();
    try {
      setCookie(res, {
        name: "Space",
        value: "Cat",
        httpOnly: true,
        secure: true,
        maxAge: -1,
      });
    } catch {
      error = true;
      assert(error);
    }

```

Reply:
Agreed, though the formatting on the suggested patch will need to be fixed.